### PR TITLE
v2v: add cases about logs when vcenter or esxi address is invalid

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.negative_test.logs.open_source.vcenter_ip.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (151.03 s)
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.negative_test.logs.open_source.esxi_ip.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (19.71 s)
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.negative_test.logs.open_source.esxi_ip.esx_67.rhev.rhv_upload.it_vddk.esx_uri: PASS (145.37 s)

**Comment by xiaodwan** [comment]
@chloerh @yantian0 @chunfuwen Please help to review this PR, thanks.

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -445,10 +445,24 @@
...
+            main_vm = VM_NAME_ESX_DEFAULT_V2V_EXAMPLE
+            variants:
+                 - open_source:
+                    checkpoint = 'invalid_source'
+                    only negative_test
+                    expect_msg = 'yes'
+                    msg_content = 'Opening the source'
+                    variants:
+                        - vcenter_ip:
+                            invalid_vpx_hostname = 1.2.3.4
```
Comment (chunfuwen) [inline v2v/tests/cfg/function_test_esx.cfg:459]: change 1.2.3.4 -->'1.2,3.4' ?

Comment (xiaodwan) [reply v2v/tests/cfg/function_test_esx.cfg]: Done

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**